src/jdk.jpackage/windows/native/libjpackage/VersionInfo.cpp

```diff
@@ -180,7 +180,7 @@ void VersionInfo::fillBuffer(std::ostream& buf) const {
     write(buf, createFIXEDFILEINFO()); // Value
     add32bitPadding(buf); // Padding2
 
-    const DWORD neutralLangId = (0x04b0 | MAKELANGID(LANG_NEUTRAL, SUBLANG_NEUTRAL) << 16);
+ // const DWORD neutralLangId = (0x04b0 | MAKELANGID(LANG_NEUTRAL, SUBLANG_NEUTRAL) << 16);
```

Comment:
As ` neutralLangId` is unused, remove it instead of commenting it out.

Reply:
Will do